app/pages/patientdata/patientdata.less

```diff
@@ -100,6 +100,15 @@
   float: right;
 }
 
+.patient-data-subnav .print-loading-spinner {
+  width: 16px !important;
```

Comment:
Yep, I felt dirty putting that in.  For some reason I thought react-loader didn't allow it to be set via props (should've looked at the gh page instead of a code example from an old project of mine ;) )